essential_reading/DataSources/TestData.zarr/.zattrs

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
are the TestData.csv and TestData.zarr files getting used somewhere?

Reply:
Yes.   The chunking example uses the zarr... and the yet-to-be-uploaded NWM benchmark uses the csv.  We can strike from the repo, but then we need to find a way to load sample data via network (S3? http?) for those demos to operate on.

Reply:
These datafiles are retained in the repo for now -- unless/until we can find a network-accessible source.   The `zarr` data file got moved to preprocessing folder.  the `csv` is in with NWM modeling, since that's where it is used.